graphstorm-processing/graphstorm_processing/graph_loaders/row_count_utils.py

```diff
@@ -85,25 +85,6 @@ def add_row_counts_to_metadata(self, metadata_dict: dict) -> dict:
         )
         self._add_counts_for_features(top_level_key="node_data", edge_or_node_type_key="node_type")
 
```

Comment:
shall we call verify_metadata_match here?

Reply:
We now do this inside distributed_executor.py, line 247.